### PR TITLE
Build Fast DDS for Windows on build_fastdds workflow

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: .github/actions/install-eprosima-dependencies-windows/action.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
+name: Install eProsima dependencies
+description: Install and setup foonatham-memory, Fast CDR and Fast DDS for linking and building applications in Windows
+inputs:
+  cmake_build_type:
+    description: 'Specifies the build type on single-configuration generators'
+    required: true
+    default: 'Release'
```
Comment (juanlofer-eprosima) [inline .github/actions/install-eprosima-dependencies-windows/action.yml:8]: NIT: Why default if mandatory?

File: .github/actions/install-eprosima-dependencies-windows/action.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        cmake --build build\fastcdr --config ${{ inputs.cmake_build_type }} --target install
+    - name: Install Fast DDS
+      shell: pwsh
+      run: >
+        git clone https://github.com/eProsima/Fast-DDS.git;
+        cmake -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}\eprosima\install `
+          -DCMAKE_PREFIX_PATH=${{ github.workspace }}\eprosima\install `
+          -DTHIRDPARTY=ON -DSECURITY=ON -DCOMPILE_EXAMPLES=OFF -DEPROSIMA_BUILD_TESTS=OFF `
+          -DINTERNAL_DEBUG=ON -Ax64 -T host=x64 -B build\fastdds Fast-DDS;
+        cmake --build build\fastdds --config ${{ inputs.cmake_build_type }} --target install
```
Comment (juanlofer-eprosima) [inline .github/actions/install-eprosima-dependencies-windows/action.yml:36]: missing EOF

**Review by juanlofer-eprosima — CHANGES_REQUESTED** [review]

File: .github/actions/install-eprosima-dependencies-windows/action.yml (hunk 1 above)
Comment (rsanchez15) [reply .github/actions/install-eprosima-dependencies-windows/action.yml:8]: I'd keep it as it is in case we miss this input parameter.

**Review by juanlofer-eprosima — APPROVED** [review]
LGTM

**Pull request merged**